lib/ace/token_iterator.js

```diff
@@ -110,6 +111,11 @@ var TokenIterator = function(session, initialRow, initialColumn) {
         return this.$rowTokens[this.$tokenIndex];
     };
 
+
+    this.getTokens = function () {
```

Comment:
vice versa, this method looks unused (I have not found any usages in this pull request). Is it really needed?